extensions/omni_httpc/omni_httpc.c

```diff
@@ -242,6 +254,8 @@ static int on_body(h2o_httpclient_t *client, const char *errstr, h2o_header_t *t
   h2o_buffer_t *buf = *client->buf;
   if (buf != NULL && buf->size > 0) {
     appendBinaryStringInfo(&req->body, buf->bytes, buf->size);
+    // TODO: hack to copy the the request body
+    req->body_iovec = h2o_iovec_init(buf->bytes, buf->size);
```

Comment:
I had an issue with the setting the value for `StringData`0, could not figure it out yet..., so I used this filed as a workaround for now.